Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/FP16_Compression.md

> # Compressing of a Model to FP16 {#openvino_docs_MO_DG_FP16_Compression}

Reviewer: ```suggestion
# Compressing a Model to FP16 {#openvino_docs_MO_DG_FP16_Compression}
```

```

> > **NOTE**: Using `--data_type FP32` will give no result and will not force `FP32` 

Reviewer: I cannot change the heading here. I think the heading should be "Compressing a Model to FP16".

Author: Done.

> precision in the model. If the model was `FP16`, it will have `FP16` precision in IR as well.

The resulting model will occupy about twice as less space in the file system, but it may have some accuracy drop.

> Still, degradation of accuracy is negligible for the majority of models. 

Reviewer 2: ```suggestion
The resulting model will occupy about half of the previous space in the file system, but lose some of its accuracy.
For most models, the accuracy drop is negligible. 
```

> Refer to the [Working with devices](../../OV_Runtime_UG/supported_plugins/Device_Plugins.md) page for details on how plugins handle compressed `FP16` models.

Reviewer: I changed the order here becausｅ　ｉｔ　ｍａｙ　ｓｏｕｎｄ　ａ　ｌｉｔｔｌｅ　ｂｉｔ　ｓｕｄｄｅｎ　ｔｏ　ａｓｋ　ｕｓｅｒｓ　ｔｏ　ｒｅｆｅｒ　ｔｏ　ａ　ｄｏｃｕｍｅｎｔ　ｃｏｎｓｉｄｅｒｉｎｇ　ｏｆ　ｔｈｅ　ｃｏｎｔｅｘｔ．

> > **NOTE**: `FP16` compression is sometimes used as initial step for `INT8` quantization.

Reviewer: ```suggestion
> **NOTE**: `FP16` compression is sometimes used as the initial step for `INT8` quantization.
```
